health-services/project/CHANGELOG.md

```
@@ -25,19 +30,22 @@ All notable changes to this module will be documented in this file.
 - Added `ExistentEntityValidator` fixes
 
 ## 1.1.2 - 2024-02-26
+
 - Implemented validation for updating project start date and end date.
 - Added numberOfSessions field in additional details for attendance registry.
 
 ## 1.1.1 - 2023-11-15
+
 - Added tag in project beneficiary 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)